_ping.php

```diff
@@ -750,7 +776,7 @@ protected function check2(): void {
     }
 
     if ($good_count < 1 && $bad_count > 0) {
-      $this->setStatus('error', 'Connection errors.', ['errors' => $msgs]);
+      $this->setStatus('warning', 'Connection errors.', ['errors' => $msgs]);
```

Comment:
We changed this to warning because ping would otherwise send 500 error although Memcache being down does not mean the site is down.